flink-libraries/flink-cep/src/test/java/org/apache/flink/cep/operator/CEPOperatorTest.java

```diff
@@ -654,6 +654,7 @@ public boolean filter(Event value) throws Exception {
 						Lists.newArrayList(startEvent2, endEvent, middleEvent3)
 				)
 		);
+		harness.close();
```

Comment:
ok, did that - would have been nice to use try-with-resources but I didn't want to change too much and an operator's `close()` method does allow throwing `Exception`s and so should `AbstractStreamOperatorTestHarness#close()` (probably)